docs/sponsors.md

```diff
@@ -2,5 +2,7 @@
 
 Sponsorship for the PyTexas Virtual Meetup's been wrangled up by these fine organizations below. We're as grateful as a longhorn in spring grass! A big Texas-sized THANK YOU to each one of 'em.
 
+(Want to see your company or organization on this list? Send a warm howdy to sponsorship@pytexas.org)
+
 ## JetBrains
-![JetBrains Logo](assets/images/jetbrains_logo.png){align=center width="300"}
\ No newline at end of file
+![JetBrains Logo](assets/images/jetbrains_logo.png){align=center width="300"}
```

Comment:
I just used the GitHub Web UI editor, so I can't even say "oh looks like my editor made this change".

WTF

Reply:
Well, you can say that...